bin/sl-deploy.js

```diff
@@ -12,11 +13,13 @@ var fs = require('fs');
 var maybeTunnel = require('strong-tunnel');
 var path = require('path');
 
-function printHelp($0, prn) {
-  var USAGE = fs.readFileSync(require.resolve('./sl-deploy.txt'), 'utf-8')
-      .replace(/%MAIN%/g, $0)
-      .trim();
+if (path.basename(process.cwd()) === 'test')
```

Comment:
sorry, not following. Why don't you always do `glb.setRootDir(path.resolve(__dirname, '..'))`, which is always the strong-deploy root dir, regardless of whatever the CWD is?